BookMyShowApp.API/Interface/ICustomer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using BookMyShow.Data;
+using BookMyShow.Data.ViewModels;
+
+namespace BookMyShowApp.API.Interface
+{
+    public interface ICustomer
+    {
+        Task<IEnumerable<Customer>> SaveCustomerDetails(CustomerModel c);
+
+        Task<IEnumerable<Customer>> GetCustomerDetails();
```

Comment:
Where is this method used? Generally in book my show app - we don't have any api that exposes all customer details right.